package.nls.json

```diff
@@ -33,6 +33,7 @@
     "azureFunctions.enableJavaRemoteDebugging": "Enable remote debugging for Java Functions Apps running on Windows. (experimental)",
     "azureFunctions.enableOutputTimestamps": "Prepends each line displayed in the output channel with a timestamp.",
     "azureFunctions.enableRemoteDebugging": "Enable remote debugging for Node.js Function Apps running on Linux App Service plans. Consumption plans are not supported. (experimental)",
+    "azureFunctions.endOfLifeWarning": "Enable End Of Life Warning",
```

Comment:
Maybe something like "Show a warning when creating Function Apps with stacks within 6 months of their end of life date."?

Reply:
Do we know if stacks that are passed their end of life date still appear? If so, may want to modify the message to "...with stacks passed or within...".

Reply:
Not sure, what about that one with an EOL of yesterday?

Reply:
I actually read it wrong since the months are zero indexed so the actual EOL for Powershell 7.0 is 12/3. Whoops lol.